### PR TITLE
Fix MTS and add T1w disc labels

### PR DESCRIPTION
**Pull request by sandrinebedard**
## Description

This PR adresses #25 . It fixes the MTS filenames to comply with BIDS.

It adds T1w manual disc labels too. (Sorry for adding this in the same branch )

### CODE REVIEW

**Comment by mguaypaq** [comment]
Sorry for the delay. But it looks like the image files for the manual disc labels are not uploaded to Amazon. Can you run the commands from Step 6 of [the upload instructions](https://github.com/spine-generic/spine-generic/wiki/git-annex#images-niigz)?

**Review by mguaypaq — CHANGES_REQUESTED** [review]
Other than the missing image, the PR looks good:
- The json sidecars for the new labels look fine.
- The MTS rename is OK.
- All the `RPI_r` suffixes got removed.
- bids-validator is happy.

The CI checks are failing because of an unrelated problem: the script is pinned to v2.3 of the spine-generic validation script, which fails to install because the dependency `sklearn` got renamed to `scikit-learn`. I'll make a new release v2.3.1 to fix this, and we can unpin the spine-generic validation script in a different PR.

**Comment by sandrinebedard** [comment]
> re failing because of an unrelated problem: the script is

oups my bad! shoul be there now!

**Comment by sandrinebedard** [comment]
I got an error when running the following command for the disc labels:

~~~
(base) sebeda@joplin:~/datasets/data-single-subject$ git annex copy --to amazon
copy derivatives/labels/sub-chibaIngenia/anat/sub-chibaIngenia_T1w_labels-disc-manual.nii.gz (checking amazon...) (HttpExceptionRequest Request {
  host                 = "data-single-subject---spine-generic---neuropoly.s3.ca-central-1.amazonaws.com"
  port                 = 443
  secure               = True
  requestHeaders       = [("Date","Wed, 21 Feb 2024 19:29:21 GMT"),("Authorization","<REDACTED>"),("X-Amz-Content-Sha256","e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"),("X-Amz-Date","20240221T192921Z")]
  path                 = "/SHA256E-s86109--c3bbcd626b43663dbc500ecc65c54756abcbb93a6162a9ad76ecb4daaa2a0258.nii.gz"
  queryString          = ""
  method               = "HEAD"
  proxy                = Nothing
  rawBody              = False
  redirectCount        = 10
  responseTimeout      = ResponseTimeoutDefault
  requestVersion       = HTTP/1.1
}

 (StatusCodeException (Response {responseStatus = Status {statusCode = 403, statusMessage = "Forbidden"}, responseVersion = HTTP/1.1, responseHeaders = [("x-amz-request-id","ZG182H0X4PCDETR6"),("x-amz-id-2","0ZVheexX8/zXEqLDsunGdBC1Z8Bu7SvYONgoEVUDbjIM2bqJA9MDGV0sbFgBQjAUgifpc3EmwAI="),("Content-Type","application/xml"),("Date","Wed, 21 Feb 2024 19:29:21 GMT"),("Server","AmazonS3")], responseBody = (), responseCookieJar = CJ {expose = []}, responseClose' = ResponseClose}) "")) failed
copy derivatives/labels/sub-douglas/anat/sub-douglas_T1w_labels-disc-manual.nii.gz (checking amazon...) (HttpExceptionRequest Request {
  host                 = "data-single-subject---spine-generic---neuropoly.s3.ca-central-1.amazonaws.com"
  port                 = 443
  secure               = True
  requestHeaders       = [("Date","Wed, 21 Feb 2024 19:29:22 GMT"),("Authorization","<REDACTED>"),("X-Amz-Content-Sha256","e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"),("X-Amz-Date","20240221T192922Z")]
  path                 = "/SHA256E-s69995--2977e303bf5f589a9a0fb9f02ce1e6d4727ab35c0d68e24769cc5979d8229bb3.nii.gz"
  queryString          = ""
  method               = "HEAD"
  proxy                = Nothing
  rawBody              = False
  redirectCount        = 10
  responseTimeout      = ResponseTimeoutDefault
  requestVersion       = HTTP/1.1
}
~~~

**Comment by sandrinebedard** [comment]
The discs should now be on amazon!

**Review by mguaypaq — APPROVED** [review]
